supervisor/utils/codenotary.py

```diff
@@ -83,7 +82,7 @@ async def cas_validate(
     if _ATTR_ERROR in data_json:
         raise CodeNotaryBackendError(data_json[_ATTR_ERROR], _LOGGER.warning)
 
-    if data_json[_ATTR_VERIFICATION][_ATTR_STATUS] == 0:
+    if data_json[_ATTR_STATUS] == 0:
```

Comment:
@pvizeli Tests were failing because it said the `verification` attribute didn't exist. I tried running the `cas authenticate` command directly in terminal with the same hash and signer and got a response like this:
```json
{
  "uid": "1644241102076801109",
  "kind": "file",
  "name": "stable.json",
  "hash": "8e15c35c6e53d216a9465eb1fecbdad214c43ab02a1f08f007820ef9436f1638",
  "size": 2105,
  "timestamp": "2022-02-07T13:38:22.076801109Z",
  "contentType": "text/plain; charset=utf-8",
  "metadata": {
    "CAS_CI_ENV": "github",
    "CI": "true",
    "GITHUB_ACTION": "__home-assistant_actions",
    "GITHUB_ACTIONS": "true",
    "GITHUB_ACTOR": "agners",
    "GITHUB_API_URL": "https://api.github.com",
    "GITHUB_BASE_REF": "",
    "GITHUB_EVENT_NAME": "push",
    "GITHUB_EVENT_PATH": "/home/runner/work/_temp/_github_workflow/event.json",
    "GITHUB_GRAPHQL_URL": "https://api.github.com/graphql",
    "GITHUB_HEAD_REF": "",
    "GITHUB_JOB": "signing",
    "GITHUB_REF": "refs/heads/master",
    "GITHUB_REPOSITORY": "home-assistant/version",
    "GITHUB_RUN_ID": "1806716161",
    "GITHUB_RUN_NUMBER": "1779",
    "GITHUB_SERVER_URL": "https://github.com",
    "GITHUB_SHA": "10f638ac9cc1b9900bca7c2794afc0d2ecbb8ffb",
    "GITHUB_WORKFLOW": "Version",
    "GITHUB_WORKSPACE": "/home/runner/work/version/version"
  },
  "signer": "bm90YXJ5QGhvbWUtYXNzaXN0YW50Lmlv",
  "revoked": "0001-01-01T00:00:00Z",
  "status": 0,
  "ledger": "PublicNotarizationLedger",
  "PublicKey": "",
  "verified": true,
  "Verbose": null
}
```
And yea, that field is missing. So it seems like maybe the underlying API changed since you made this? I was looking in the CAS repo and not really seeing documentation detailing this JSON or an API spec so its hard to tell specifically what changed when. Is there a spec somewhere that I'm failing to find I can use to confirm this is the correct field? Based on the response I was torn between making this condition based on `status` or `verified`. `verified` sounds right but `status` has a `0` which matches the current check so I went with that.

Reply:
Make sense, we can later also use the data from the ledger to verify it a bit better. But for now it should be fine